templates/Alloy.Mvc/Views/StartPage/Index.cshtml

```diff
@@ -1,5 +1,7 @@
-@model PageViewModel<Alloy.Mvc._1.Models.Pages.StartPage>
+@inherits Alloy.Mvc._1.Views.CustomContentAreaItemCssRenderingBase<PageViewModel<Alloy.Mvc._1.Models.Pages.StartPage>>
```

Comment:
This seems like a weird base class to inherit from. Wouldn't it make more sense that all views inherit the same base class, and that class has the `OnItemRendered` method? Or just add `OnItemRendered` to `PageViewModel`?